### PR TITLE
First GHA workflow

### PR DESCRIPTION
**Pull request by ksbeattie**
An very initial set-up of GitHub Actions, to run pytest.

This installs idaes-pse from the nawi-hub/idaes-pse fork.  More needs to be done around all this, but this is at least a starting point.

## Fixes/Addresses:

Addresses #31 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,6 +50,8 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
+pytest_output/
+pytest_output_debug/
```
Comment (bknueven) [inline .gitignore:54]: Once PR #87 is merged we shouldn't need these lines.

**Review by bknueven — APPROVED** [review]
Seem reasonable. I'm not sure why this isn't firing now, but we should probably merge and see if it works.

I'll add an issue to get mpi4py tests running.

File: .gitignore (hunk 1 above)
Comment (ksbeattie) [reply .gitignore]: I've removed them.  Thanks.

**Review by andrewlee94 — APPROVED** [review]
I don't know too much about this, but nothing jumps out as wrong.

We should set up our own IDAES fork at some point so we can control our version through tags.

**Comment by ksbeattie** [comment]
> Seem reasonable. I'm not sure why this isn't firing now, but we should probably merge and see if it works

I think that's because this is the first one.  I tested it in my branch, with different settings.  But we'll see in a moment...

**Comment by ksbeattie** [comment]
> I don't know too much about this, but nothing jumps out as wrong.
> 
> We should set up our own IDAES fork at some point so we can control our version through tags.

I've got the fork set up, and it's being used here, but finer control via tags is still to come.

**Pull request merged**